### PR TITLE
[ACR] Populate Task Timer Trigger in Run Response

### PR DESCRIPTION
**Pull request by jaysterp**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by jaysterp** [comment]
//cc @northtyphoon

**Review by northtyphoon — APPROVED** [review]
LGTM

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-04-01/containerregistry_build.json [hunk 1]
```diff
@@ -793,6 +799,10 @@
           "$ref": "#/definitions/SourceTriggerDescriptor",
           "description": "The source trigger that caused the run."
         },
+        "timerTrigger": {
```
Comment (sergey-shandar) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-04-01/containerregistry_build.json]: Could we add new properties at the end of `"properties": { … }` ?

**Review by sergey-shandar — CHANGES_REQUESTED** [review]

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-04-01/containerregistry_build.json (hunk 1 above)
Comment (jaysterp) [reply specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-04-01/containerregistry_build.json]: Sure. See update

**Comment by mnltejaswini** [comment]
@sergey-shandar  The model validation complaints about the types that can have polymorphic types based on the discriminator value. The discriminator type is readonly. From SDK perspective,  users will never set the type, instead they create  specific polymorphic type which will automatically set the type property. I guess the model validation should skip for these discriminator type values or please advise if there are any recommendations for such polymorphic types.

**Comment by jaysterp** [comment]
@sergey-shandar Do you have any recommendations on how to fix this model validation issue?

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-04-01/containerregistry_build.json [hunk 2]
```diff
@@ -817,6 +823,10 @@
           "type": "string",
           "readOnly": true
         },
+        "updateTriggerToken": {
```
Comment (sergey-shandar) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-04-01/containerregistry_build.json]: Could we move this property at the end as well?

Comment (jaysterp) [reply specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-04-01/containerregistry_build.json]: Sure, will do.

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**